src/test_l1_hdmi_cec_driver.c

```diff
@@ -832,7 +736,7 @@ void test_hdmicec_hal_l1_removeLogicalAddress_negative( void )
  */
 void test_hdmicec_hal_l1_removeLogicalAddress_positive( void )
 {
-    int result;
+    HDMI_CEC_STATUS result = HDMI_CEC_IO_SUCCESS;;
```

Comment:
In the description fo this function, just make it clear that it is only for sink devices and not for the source devices.

Reply:
It is already mentioned in the Test Description.

Reply:
Like in line 597, please mention the same around line 673.  The statement is " This test case is only applicable for sink devices"

Reply:
Addressed